app/containers/main-container.jsx

```diff
@@ -50,7 +55,9 @@ const mapStateToProps = state => ({
 
 const mapDispatchToProps = dispatch => ({
   stepAddGoal: () => { dispatch(action_stepAddGoal()) },
-  onInputGoal: (text) => { dispatch(action_inputGoal(text)) },
+  onInputGoal: (text) => { dispatch(actionsAddNewGoal.inputGoal(text)) },
```

Comment:
No!

You can just pass actionsAddNewGoal as the second argument to the connect function (redux will sort the rest!)